.gitignore

```diff
@@ -34,8 +34,8 @@ assets/
 # cypress
 cypress/fixtures
 cypress/screenshots
-cypress/support
 cypress/videos
+cypress.env.json
```

Comment:
This is being added – I've moved my env vars used for testing to a separate file (`cypress.env.json`) rather than part of `.env`. This is a convenience thing, just so I have more logical separation between env vars used for running the app, and env vars just used for running the tests

Reply:
Sorry, I realised I highlighted the wrong diff 😅 (it looks like we're removing `cypress/support`)

Reply:
Oh yeah, `cypress/support` is where the code for `cy.visitAs` lives, so needs to be in the repo 😄